pixi.toml

```diff
@@ -45,20 +45,25 @@ libc = "2.28"
 
 [activation]
 
-[activation.env]
-# Put pixi builds in its own build dir, otherwise `pixi run cargo build`
-# conflicts with vanilla`cargo build`, and running one invalidates (parts) of the other.
-# See https://github.com/rerun-io/rerun/issues/10934 for more.
-CARGO_TARGET_DIR = "target_pixi"
-
 [target.unix.activation.env]
 # The executable extension for binaries on the current platform.
 EXECUTABLE_EXTENSION = ""
 
+# Put pixi builds in its own build dir, otherwise `pixi run cargo build`
+# conflicts with vanilla`cargo build`, and running one invalidates (parts) of the other.
+# See https://github.com/rerun-io/rerun/issues/10934 for more.
+#
+# As of pixi 0.53.0, environment variable resolving doesn't work with this on Windows,
+# which is why we have to use the `%` syntax for Windows instead of `$`.
```

Comment:
Sounds like a bug in issue? Is there a pixi issue we can link to here from a TODO?